File: lit-udr/lit-udr-app/src/model/ReviewDto.js
export default class ReviewDto {
    constructor(Token,Id,Result,ProcessDefinitionId,ProcessInstanceId,Comment) {
      this.token = Token;
      this.id = Id;
      this.Result = Result;
      this.processDefinitionId = ProcessDefinitionId;
      this.processInstanceId = ProcessInstanceId
      this.comment = Comment;
    }
  }

File: lit-udr/lit-udr-BE/lit-udr/Model/ReviewDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace lit_udr.Model
{
    public class ReviewDto
    {
        public string Token { get; set; }
        public int Id { get; set; }
        public string Result { get; set; }
        public string ProcessDefinitionId { get; set; }
        public string ProcessInstanceId { get; set; }
        public string Comment { get; set; }
    }
}
